### DISCUSSION TITLE
Quick challenge for something i bet ya'll can't run (well rn.)

### DISCUSSION THREAD

**Discussion by Selvenity** [wheremyfoodat/Panda3DS#392] [General]
Bet ya'll can't run Dr. Mario Express.

**Comment by Selvenity** [comment]
Put a DSiWare title since it was one supported for the 3DS

**Reply by Adamillo** [reply]
Panda3DS will not have DS/DSi support because they use very different hardware compared to the 3DS. The reason the 3DS has backwards compatibility is because they literally just put DSi hardware inside it

**Reply by Selvenity** [reply]
_Everyone looks at this emulator with shame._

**Comment by wheremyfoodat** [comment]
![image](https://github.com/wheremyfoodat/Panda3DS/assets/44909372/8ee5ce32-2221-413d-a474-5b4cc4829f87)

**Reply by fallingSparr0w** [reply]
ifunny.com
